.circleci/config.yml

```diff
@@ -53,11 +53,11 @@ jobs:
           command: |
             . venv/bin/activate
             pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example
+            npm run build
             python run.py -t &
             sleep 5
             nosetests
             npm run lint
-            npm run build
```

Comment:
I had to change the place where npm run build is called because I'm using the path to some files that have the hash generated by: https://github.com/webcompat/webcompat.com/blob/0133d3dfc2070dda72979b199dff30af1074806c/webcompat/helpers.py#L46-L52
Hope it's not a problem. CircleCI was breaking because it tried to run python before generating the files I needed

Reply:
@marimeireles Is this working?

```yaml
      - run:
          name: run tests
          command: |
            # install
            . venv/bin/activate
            pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example
            # python unittests. Do not require a server
            nosetests
            # creating a build
            npm run lint
            npm run build
            # starting the test server
            python run.py -t &
            sleep 5
            # running the functional JS tests
            node --max-old-space-size=8192 ./tests/functional/_intern.js --reporters="runner" --firefoxBinary=`which firefox`
            firefox --version
```

Reply:
I think it is.
Tests are running fine with this change.

Reply:
Yes.
As long as `npm run build` runs before the `python run.py` thing I think it's going to work.

Reply:
ok it's more logical this way. Because nosetests doesn't need the server to be started.

Reply:
Ok. I'm going to change it then.
Thank you for this, Karl. :)

Reply:
But apparently I can't add your comments because they're in the command session.

Reply:
note to myself: the `cache_dict` used in `bust_cache` is not global but per app. So I guess we create multiple ones, one  for each app being launched.